manifest.json

```diff
@@ -24,7 +24,7 @@
 		"email": "maniackc_dev@crudelis.fr"
 	}],
     "requirements": {
-        "yunohost": ">= 4.1.2"
+        "yunohost": ">= 4.3.0"
```

Comment:
Note that this change is probably what's causing the CI to fail, as if we require yunohost >= 4.2, this implicitly triggers some `chmod/chown` during `ynh_setup_source`

We probably to tweak permissions (eg allow www-data probably?) to fix it